src/libhirte/common/cfg.c

```diff
@@ -137,6 +141,32 @@ int cfg_load_from_file(struct config *config, const char *config_file) {
         return ini_parse(config_file, parsing_handler, config);
 }
 
+int is_file_name_ending_with_conf(const struct dirent *entry) {
+        if (strstr(entry->d_name, ".conf") != NULL) {
+                return true;
+        }
+        return false;
+}
```

Comment:
```suggestion
bool is_file_name_ending_with_conf(const struct dirent *entry) {
        return strstr(entry->d_name, ".conf") != NULL;
}
```
Sorry for being, but could you change it like this? This way its a bit more concise.

Reply:
I think the method needs to return `int` to be usable as a filter in scandir:

```
int scandir(const char *restrict dirp,
                   struct dirent ***restrict namelist,
                   int (*filter)(const struct dirent *),
                   int (*compar)(const struct dirent **,
                                 const struct dirent **));
```

Reply:
You are right. Haven't seen the (changed?) usage of this function.